neuron/src/lib/Neuron/Zettelkasten/Zettel.hs

```diff
@@ -123,6 +124,12 @@ zettelJson Zettel {..} =
     "day" .= zettelDay
   ]
 
+zettelJsonFull :: forall a. KeyValue a => FilePath -> Zettel -> [a]
```

Comment:
This function should remain where it is used, which is `Query.hs`, and not be exported.